clients/venice-push-job/src/test/java/com/linkedin/venice/hadoop/VenicePushJobTest.java

```diff
@@ -703,7 +703,7 @@ public void testTargetedRegionPushPostValidationConsumptionForBatchStore() throw
     }
   }
 
-  @Test
+  @Test(enabled = false)
```

Comment:
nit: can you please add TODO for these 2 tests got commented out? I just want to make sure we can add them back when hybrid stores are supported, otherwise we may forget them forever

Reply:
Sure